website/docs/learning-and-resources/how-to-guides/how-to-use-git-flow-for-different-environments.md

```diff
@@ -0,0 +1,35 @@
+# How to use Git flow for different environments
```

Comment:
Just as a heads-up, this new title might break the convention of the other How-To guide pages -- most (except for a small handful?) use the "How To ..." format.

I'm not sure whether we're intending to move away from that, but I figured I'd mention it just in case.

For the time being, I used a compromise of "How To Use Git To Create Different Environments" just to keep the review moving; I'm happy with either way, just let me know

Reply:
Yeah i think its ok we should remove the 'How to' for the newer ones we are adding, because the section is already titled How to Guide. Again in the subpages, it looks odd to mention it over an over again